### ISSUE TITLE
[Feature Request] Limit meta-data auto detection of galleries to specific languages

### ISSUE THREAD

**Issue by Black-Light** [seanegoodwin/PandaViewer#6]
This could help make the auto-detection more accurate, I would like to see something akin to the settings page on exh for displaying galleries but anything will do.

**Comment by seanegoodwin** [comment]
I'm not entirely sure how ex language exclusion works, but I think if you were to exclude any languages you don't want on your ex settings page, we might filter them out as well. 

If not, this is probably worth adding.

**Comment by Triskov** [comment]
**I'm not entirely sure how ex language exclusion works, but I think if you were to exclude any languages you don't want on your ex settings page, we might filter them out as well.**

From what I've seen, all the exhentai settings are browser specific, rather than account. 
When I ran the metadata collection, even though I have it set it only see Japanese and English in the results, it used metadata from whatever it could find.

**Comment by seanegoodwin** [comment]
Yeah, it does seem like that now. Makes sense. I'll see about implementing this.

**Comment by Black-Light** [comment]
Yeah Ex just uses the **uconfig** cookie understanding how this works is going to be the key.

for now just Importing your own cookie content similar to how the pw hash works would be fine until / if you can work out how to edit the cookie so people can set this in the program. This way you can still set the settings how you want them for PandaViewer on exh paste that cookie content into PandaViewer then you can change the settings on the site to something else if you want to have different settings for browsing.
